### PR TITLE
global: initial alembic recipes

### PR DESCRIPTION
**Pull request by jirikuncar**
Depends on inveniosoftware/invenio-db#59

### CODE REVIEW

**Comment by jirikuncar** [comment]
LGTM (cc @lnielsen)

File: invenio_records/alembic/1095cdf9f350_create_records_branch.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+"""Create records branch."""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1095cdf9f350'
+down_revision = None
+branch_labels = (u'invenio_records',)
```
Comment (tiborsimko) [inline invenio_records/alembic/1095cdf9f350_create_records_branch.py]: Thanks for changing the branch prefixes. Looks good to me, CC @lnielsen.

File: setup.py [hunk 2]
```diff
@@ -79,12 +79,13 @@
...
 install_requires = [
     'blinker>=1.4',
-    'flask-celeryext>=0.1.0',
+    'flask-celeryext>=0.2.2',
     'Flask>=0.11.1',
     'jsonpatch>=1.11',
     'jsonresolver>=0.1.0',
     'jsonref>=0.1',
     'jsonschema>=2.5.1',
+    'kombu>=3.0.37,<3.1',  # FIXME remove when Flask-CeleryExt is fixed
```
Comment (lnielsen) [inline setup.py]: Is this still needed after Flask-CeleryExt 0.2.2?

**Comment by lnielsen** [comment]
LGTM, just check the need for kombu

File: setup.py (hunk 2 above)
Comment (jirikuncar) [inline setup.py]: It was failing yesterday even with the upgraded Flask-CeleryExt. I have removed it and let's see now.

**Comment by jirikuncar** [comment]
:shipit:

**Pull request merged**